src/pages/home/components/feature-information/feature.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { spaces, breakpoints, colors, textStyles } from '../../../../styles'
+import styled from 'styled-components'
+
+
+const MyDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`
+
+const Image = styled.img`
+width: ${spaces.relax * 5}px;
+align-self: center;
+`
+
+const Title = styled.h4`
+${textStyles.h4Bold}
+`
+
+const MyLink = styled.p `
```

Comment:
this should be a `Link` not a `p`, remember that clicking in this link the user will be redirected to another page.